### PR TITLE
feat: Add a `x/wasm` module

### PR DESCRIPTION
**Pull request by youngjoon-lee**
# CAUTION!!!!
Merge this PR directly to the `master`, after the `stargate` branch is merged to the `master`.
It's because this PR is not the part of the issue #108.

# Changes
The Starport has a command: [`starport module import wasm`](https://github.com/tendermint/starport/pull/642), but unfortunately, it doesn't work well. Also, it's based on the old CosmWasm v0.14.

So, I imported CosmWasm v0.17 manually by referring to https://github.com/CosmWasm/wasmd/blob/v0.17.0/app/app.go

I tested it manually using the [Escrow](https://docs.cosmwasm.com/0.14/getting-started/compile-contract.html) example.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -15,8 +16,8 @@ require (
 	github.com/onsi/ginkgo v1.15.0 // indirect
 	github.com/onsi/gomega v1.10.5 // indirect
 	github.com/pborman/uuid v1.2.0
+	github.com/prometheus/client_golang v1.10.0
```
Comment (gyuguen) [inline go.mod:19]: I remember `prometheus` is monitoring tools.
Is used here code?
or auto importing this?

Comment (youngjoon-lee) [reply go.mod:19]: Yeah. The Stargate supports the [Telemetry](https://docs.cosmos.network/master/core/telemetry.html) feature which collects some metrics to Prometheus, even though I haven't read it seriously.

`x/wasm` modules have some telemetry-related options: https://github.com/CosmWasm/wasmd/blob/d2e35c249e39e90c5a85e03e0b75800431612b20/cmd/wasmd/root.go#L205

Comment (gyuguen) [reply go.mod:19]: OK. I got it!

**Review by gyuguen — APPROVED** [review]
👍

**Pull request merged**